STYLE.md

```diff
@@ -134,6 +134,30 @@ This section describes guidelines for designing public APIs. It can also be appl
    }
    ```
 
+1. Use options as arguments to functions and methods, rather than positional arguments.
```

Comment:
```suggestion
1. When there is a significant number of arguments to a function or method, prefer to use a single options object as the argument, rather than many positional arguments.
```